### PR TITLE
RFC 009 - Transparent code bases

### PR DESCRIPTION
**Pull request by craigjbass**
Proposal for transparent code bases.

### CODE REVIEW

File: rfc-009-transparent-code-bases.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+It is currently difficult for teams to see what other teams are working on, and gain insights, provide feedback and generally for us as a company to learn from each other.
+
+Even in the case where projects are hosted on madetech github, it's hard to filter through the large number of repositories.
+
+## Proposal
+
+Teams must publish, to a centralised location, links to details on how any Made Tech engineer can access their customer code.
+
+In situations where NDAs preclude providing transparent access to codebases, there must be a document that outlines this and reasons why.
```
Comment (danielburnley) [inline rfc-009-transparent-code-bases.md]: What if it isn't even NDA reasoning, in cases of the customer just not wanting to add more than the necessary people to the company GH account

Comment (craigjbass) [reply rfc-009-transparent-code-bases.md]: Have included a potential proposal around this on line https://github.com/madetech/rfcs/pull/12/files#diff-96afec47c53e5e2879b8cd927d3a2cfcR20 but unsure how feasible this actually is

Comment (craigjbass) [reply rfc-009-transparent-code-bases.md]: (Other ideas welcomed)

File: rfc-009-transparent-code-bases.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Summary
+
+Teams are expected to publish details on accessing their source code.
+
+## Problem
+
+It is currently difficult for teams to see what other teams are working on, and gain insights, provide feedback and generally for us as a company to learn from each other.
+
+Even in the case where projects are hosted on madetech github, it's hard to filter through the large number of repositories.
```
Comment (camillavk) [inline rfc-009-transparent-code-bases.md]: How will having more transparency (and potentially even more forked repositories) help with this? It is already hard to filter through the large number of them, but I'm not sure how having links to what I assume would be very similar to the readmes in a centralised location would make this easier to navigate?

Comment (craigjbass) [reply rfc-009-transparent-code-bases.md]: Google Spreadsheet:

| Customer | Link to readme |
|---|---|
| Akzo eCom | click here |
| Akzo eFulfilment Hub | click here |

Comment (craigjbass) [reply rfc-009-transparent-code-bases.md]: I guess to answer the question, I'm less concerned about our github being full of repos, more being able to get signal from the noise.

File: rfc-009-transparent-code-bases.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Proposal
+
+Teams must publish, to a centralised location, links to details on how any Made Tech engineer can access their customer code.
```
Comment (danielburnley) [inline rfc-009-transparent-code-bases.md]: Where would we define this centralised location, the handbook, here, elsewhere?

Comment (craigjbass) [reply rfc-009-transparent-code-bases.md]: Potentially a sensitive list so perhaps a link to a private spreadsheet (shared with Made Tech org) from the handbook?

File: rfc-009-transparent-code-bases.md (hunk 2 above)
Comment (chrisblackburn) [inline rfc-009-transparent-code-bases.md]: Made Tech
GitHub

**Comment by cybojenix** [comment]
This RFC has gone quiet, does someone wish to continue it on?

**Pull request closed without merging**